### ISSUE TITLE
Will other samplers be supported in the future?	

### ISSUE THREAD

**Issue by IDKiro** [XPixelGroup/DiffBIR#48]
I noticed that you've removed the DDIM option in the current version of the code, even though it didn't seem to work in the initial version. Sampling efficiency is one of the obstacles to practicality, especially for high-resolution images. Will DDIM and DPM-Solver samplers be supported in the future?

**Comment by IDKiro** [comment]
I conducted experiments applying DDIM and DPM-Solver and found that they performed very poorly. After debugging, I found that ODE-based samplers inherently couldn't reconstruct images similar to the conditional images. I tried setting the eta of DDIM to 1.0, and obtained results similar to those from the Spaced Sampler. Did you also remove DDIM based on similar experimental observations?

**Comment by 0x3f3f3f3fun** [comment]
Yes! We find that the DDIM sampler will produce unpleasant results and we don't know what the reason is.
